apps/elasticsearch/vars/all/main.yml

```diff
@@ -50,3 +50,11 @@ elasticsearch_index_node_left_delayed_timeout: "30s"
 # (see index.store.type in ES documentation)
 elasticsearch_index_store_type: "niofs"
 
+# Security settings
+elasticsearch_security_enabled: false
+elasticsearch_bootstrap_volume_size: 2Mi
```

Comment:
This will not work and create a volume of 1GB.

Reply:
So basically it's a gluster-related issue. Should we ignore this then? Should I add a comment?

Reply:
Yes you should add a comment and set to `1Gi` I think